### DISCUSSION TITLE
How to use the MPV OSD stats (via shortcut `Shift + i`) without an external MPV window?

### DISCUSSION THREAD

**Discussion by polarathene** [smplayer-dev/smplayer#405] [Q&A] [answered]
I noticed this same shortcut produces similar results (_at least using the MPV backend, I haven't tried with MPlayer_), but not the same as when I configure for a separate MPV window? (_which has better stats_)

Is this being produced from SMPlayer instead? Or is there a way to get the same stats output as MPV itself presents? Notably with `Shift + i` you'll get some extras such as history graphing for some stats, whereas shortcut `i` lacks that but still provides brief stats overlay and SMPlayer instead presents time info.

Is this more to do with a difference of shortcuts being relevant for SMPlayer vs MPV's own shortcuts when it has it's own window? Can I configure SMPlayer to show these stats instead via shortcut? I am aware that [MPV uses a lua script for these stats](https://mpv.io/manual/stable/#stats), so perhaps I just need to map a shortcut to that?

MPV [integrated this lua script](https://github.com/Argon-/mpv-stats/) ([current version in MPV codebase](https://github.com/mpv-player/mpv/blob/master/player/lua/stats.lua)).

**Comment by smplayer-dev** [comment, marked as answer]
This will be fixed in the next version.

**Comment by smplayer-dev** [comment]
Implemented in https://github.com/smplayer-dev/smplayer/commit/3c422f9a7e50909c08dcd908057346e9046db3ec
